tools/publisher/src/package.rs

```diff
@@ -254,6 +289,11 @@ fn batch_packages(packages: Vec<Package>) -> Result<Vec<PackageBatch>> {
         break;
     }
 
+    // Sort packages within batches so that `--continue-from` work consistently
+    for batch in batches.iter_mut() {
+        batch.sort_by(|a, b| a.handle.cmp(&b.handle));
+    }
+
     // Push the final batch
     if !packages.is_empty() {
         batches.push(packages);
```

Comment:
I think this is a bug? we aren't sorting the final batch

Reply:
Great catch! Fixed and added a test case that repros it.